### DISCUSSION TITLE
Windows Terminal Preview v1.13.10395.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#12477] [Announcements]
This is a quick servicing release to fix a couple of big issues in the initial 1.12 stable drop.

As with the previous 1.13 release, it is shipping alongside a 1.12 release for anybody who is stuck on 19H1, 19H2 or 19H3.

## Bug Fixes

* Terminal is once again localized (#12375)
   * We encountered an issue in automatic language detection, which resulted in our package's languages
     being ignored.
* "Default Terminal" profile matching now works better for profiles containing unquoted whitespace (#12348)
* We believe we have fixed an unusual crash on launch in constructing the taskbar Jump List (#12430)
* Symbols are now published to the public symbol server! Woot! (#12441)
* We've fixed a crash that used to surface while Magnifier or other assistive technologies were running (#12436)
* Ubuntu users with overridden commandlines will no longer see a confusing "`~` could not be found" error (#12437)
* Toggle switches in the settings UI now fit longer languages (like Polish) (#12381)
* More of the settings UI is centered horizontally (#12374)
* The breadcrumbs have been picked up and will no longer navigate you to strange cottages (#12376)
* The color schemes page no longer has a strange focus rectangle (#12439)
* `vifm` can no longer trigger a race condition in cursor visibility (#12434)
* The reset arrow icon is no longer a strange box containing only "hope" (#12438)
* OSC `9;9` with an empty path will no longer send Terminal to a farm upstate (#12432)
* When Terminal is set as your default terminal, autoelevation will no longer cause all handoffs to crash (#12442)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.13.10395.0'>Windows Terminal Preview v1.13.10395.0</a>.</em>

**Comment by sba923** [comment]
19H3? 🤔

**Reply by zadjii-msft** [reply]
that was a joke 😉
